tests/installation/await_install.pm

```diff
@@ -125,6 +125,13 @@ sub run {
     }
     # on s390 we might need to install additional packages depending on the installation method
     if (check_var('ARCH', 's390x')) {
+      #For poo#93949, we need password possibility
+      if (!is_sle && !is_leap) {
```

Comment:
Not imported FWICT

Reply:
What do you mean with "not imported"? That is a test of whether Tumbleweed is used.

Reply:
Leap and SLE are using the old method.

Reply:
Needs `use version_utils qw(:VERSION :BACKEND is_sle is_leap);` at the top

Reply:
Done